pilot/pkg/kube/inject/mesh.go

```diff
@@ -17,220 +17,4 @@ package inject
 const (
 	sidecarTemplateDelimBegin = "[["
```

Comment:
This whole `mesh.go` file can be deleted, isn't it?

Reply:
Done.